docs/source/advanced/bgfx.rst

```diff
@@ -114,6 +114,6 @@ Tweaking BGFX HLSL Settings inside MAME
 
 Start by loading MAME with the game of your choice (e.g. **mame pacman**)
 
-The tilde key (**~**) brings up the on-screen display options. Use up and down to go through the various settings, while left and right will allow you to change that setting. Results will be shown in real time as you're changing these settings.
+The tilde key (``~``) brings up the on-screen display options. Use up and down to go through the various settings, while left and right will allow you to change that setting. Results will be shown in real time as you're changing these settings.
```

Comment:
This is a key label, not a filename or file fragment.  Bold is probably better than fixed pitch for this.